bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/Image.java

```diff
@@ -316,23 +323,13 @@ default String getSrcset() {
     }
 
     /**
-     * Returns the value for the {@code width} html attribute of the image.
-     *
-     * @return the value for the image's {@code width} attribute, if one was set, or {@code null}.
-     * @since com.adobe.cq.wcm.core.components.models 12.21.0;
-     */
-    default String getWidth() {
-        return null;
-    }
-
-    /**
-     * Returns the value for the {@code height} html attribute of the image.
+     * Returns an array with the base image width and height.
      *
-     * @return the value for the image's {@code height} attribute, if one was set, or {@code null}.
-     * @since com.adobe.cq.wcm.core.components.models 12.21.0;
+     * @return an array with the base image width and height, if image exists, or {@code new String[] {null, null}}.
+     * @since com.adobe.cq.wcm.core.components.models 13.0.0;
      */
-    default String getHeight() {
-        return null;
+    default String[] getBaseImageResolution() {
```

Comment:
Shouldn't the method name reflect that it returns the image dimensions? E.g. `getBaseImageDimension`.